### DISCUSSION TITLE
Is my PRO-HD bricked?

### DISCUSSION THREAD

**Discussion by shoobydoodoo** [GerryDazoo/Slinger#564] [Q&A]
I had 2 PRO-HDs on the same LAN. I made sure to unplug both of them before N9. After a while of monitoring the situation, I got around to setting them up recently. I unplugged the router from the internet, then I powered on the ProHDs to set up port block rules. The first one was configured successfully and is now running without issue. The strange thing is, when I plugged in the second ProHD, the IP and MAC didn't populate my DHCP list, unlike the first ProHD. So I manually set up a static IP based on the its MAC found on the bottom sticker. Then I followed the same block rules based on static IP and MAC. Afterward, I reconnect the internet cable to the router and powered back on.

Slinger is able to connect and stream the first ProHD without issue. But for the second ProHD, it's able to connect but unable to start stream. Another strange thing is, it detected an IP address of the Slingbox that was different than the static IP that I set. It returned with the following response lines, with one line mentioning it's possibly bricked:

Version : 4.01 Running on Windows-10-10.0.19045-SP0 pid= 10596 C:\Users\JAMES.DESKTOP-PBLILDI\Videos\Slingbox\slingbox_server.exe

Using config file config.ini

Connection Manager Running on port 65434 with 10 max streams using URL slingbox.

Building page for Slingbox

Using built in default remote page definition.

Starting Streamer Thread for Slingbox

Streamer Running: 10 config.ini SLINGBOX 65434 65434 8388608

starting up on port 65434 8388608

65434 Checking for slingbox at ('[192.168.1.254](https://192.168.1.254/)', 5002) 0

65434 ('[192.168.1.254](https://192.168.1.254/)', 5002) OK

65434 Using slingbox at ('[192.168.1.254](https://192.168.1.254/)', 5002)

Streamer: 65434 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox

* Serving Flask app 'slingbox_server'

* Debug mode: off

Broadcast Responder Running

Server IP address is [192.168.1.104](https://192.168.1.104/)

Received GETSERVERCONFIG Request b'GETSERVERCONFIG' ('[192.168.1.104](https://192.168.1.104/)', 65247)

ServerConfig port=65434,urlbase=slingbox|

GETSLINGBOXIDS Request

03/22/2023, 18:01:14.250 Streaming request from ('[192.168.1.104](https://192.168.1.104/)', 51247) /slingbox

STREAM=192.168.1.104:51247:0

Slinginfo "Solo/Pro/ProHD" 12 30 [192.168.1.254](https://192.168.1.254/) 5002 3000 10 78904321

65434 Starting Stream for [192.168.1.104:51247](https://192.168.1.104:51247/)

65434 Opening Control stream 0x2000 0 0

Connecting... ('[192.168.1.254](https://192.168.1.254/)', 5002) Control

cmd: 0x67 err: 0x2b 8

65434 Error Starting Session. Slingbox might be Bricked

65434 ERROR: Slingbox session startup failed.

65434 Logging Out

Streamer: 65434 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox

03/22/2023, 18:02:53.692 Streaming request from ('[192.168.1.104](https://192.168.1.104/)', 51251) /slingbox

STREAM=192.168.1.104:51251:0

I have a feeling it's most likely bricked. What other signs indicate it? Just wanted to come here and make sure. Thanks everyone.

**Comment by mike-uk** [comment]
Have you tried this guide to reset the HD-Pro?

[Slingbox ProHD PC Unbricking-Setup-Network-Guide.pdf](https://github.com/GerryDazoo/Slinger/files/11069720/Slingbox.ProHD.PC.Unbricking-Setup-Network-Guide.pdf)

**Reply by shoobydoodoo** [reply]
Hello Mike! this is an old post but for some reason, I just realized I never replied. Sorry for the long delayed response. I haven't had chance to reset it yet, but I will soon. Thanks for the suggestion!
